<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LinkDialog</class>
 <widget class="QDialog" name="LinkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>256</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>256</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>12</y>
     <width>251</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="timelabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Capacity (bytes/ms):</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLineEdit" name="capacity">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="port1Name">
      <property name="text">
       <string>Port 1 type:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="port1Type">
      <item>
       <property name="text">
        <string>FIFO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prioritized</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="port2Name">
      <property name="text">
       <string>Port 2 type:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="port2Type">
      <item>
       <property name="text">
        <string>FIFO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Prioritized</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="OK">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OK</sender>
   <signal>clicked()</signal>
   <receiver>LinkDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>LinkDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
